nautobot/core/settings.py

```diff
@@ -76,7 +76,7 @@
 HTTP_PROXIES = None
 JOBS_ROOT = os.getenv("NAUTOBOT_JOBS_ROOT", os.path.join(NAUTOBOT_ROOT, "jobs").rstrip("/"))
 MAINTENANCE_MODE = False
-
+AVAILABLE_PREFIX_TIMEOUT = int(os.getenv("NAUTOBOT_AVAILABLE_PREFIX_TIMEOUT", 600))
```

Comment:
pylint will complain that we're defaulting an environment variable to an int rather than a str.

```suggestion
AVAILABLE_PREFIX_TIMEOUT = int(os.getenv("NAUTOBOT_AVAILABLE_PREFIX_TIMEOUT", "600"))
```